app-multimedia/feishin/autobuild/arm64/build

```diff
@@ -0,0 +1,15 @@
+abinfo "Installing NPM dependencies ..."
+npm install --legacy-peer-deps
+
+abinfo "Building Feishin ..."
+npm run postinstall
+npm run build
+npm exec electron-builder -- --linux
+
+abinfo "Installing Feishin ..."
+install -dvm755 "$PKGDIR"/usr/lib/feishin
+cp -rv "$SRCDIR"/release/build/linux-arm64-unpacked/* "$PKGDIR"/usr/lib/feishin/
```

Comment:
You could have merged the two build scrpits by using `"$SRCDIR"/release/build/linux*-unpacked/*`.